package.json

```diff
@@ -3,8 +3,10 @@
   "version": "1.0.0",
   "description": "",
   "main": "index.js",
+  "typings": "./typings/index.d.ts",
```

Comment:
Shouldn't this be `types` instead of `typings`? See http://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html

Reply:
I have some libraries that use typings seems like they have changed this but the old version is still supported. Anyway, I have renamed it to "types".